src/assets/src/hooks/useWebSocket.ts

```diff
@@ -8,13 +8,12 @@ interface OfficeHoursMessage<T> {
 
 export const useWebSocket = <T>(url: string, onUpdate: (content: T) => void, onDelete?: (setError: (React.Dispatch<React.SetStateAction<Error | undefined>>)) => void) => {
     const [error, setError] = useState(undefined as Error | undefined);
+    const numRetries = 3;
```

Comment:
```suggestion
    const numRetries = 10;
```

I noticed the three retries exhaust pretty quickly. It might be good to allot more time for network interruptions.